deployment/aws/variable.tf

```diff
@@ -32,7 +33,7 @@ variable "pravega_num" {
 variable "pravega_aws_amis" {
   default = {
     us-east-1 = "ami-7747d01e"
-    us-west-1 = "ami-2afbde4a"
+    us-west-1 = "ami-73f7da13"
```

Comment:
This is the public ubuntu image we use. The previous one is not available any more, so I choose another alternative